pkg/api/handlers/compat/images_build.go

```diff
@@ -581,9 +581,9 @@ func BuildImage(w http.ResponseWriter, r *http.Request) {
 	} else {
 		if _, found := r.URL.Query()["pull"]; found {
 			switch strings.ToLower(query.Pull) {
-			case "false":
+			case "0", "f", "F", "false", "FALSE", "False":
```

Comment:
We already run the string through `ToLower` so no need for `F` and `FALSE` and similar